### PR TITLE
r.in.gdal: rename option location to project

### PR DESCRIPTION
**Pull request by petrasovaa**
Includes changes in user visible strings in r.in.gdal and r.import.

### CODE REVIEW

**Review by nilason — DISMISSED** [review]

**Review by nilason — APPROVED** [review]

File: raster/r.in.gdal/main.c [hunk 1]
```diff
@@ -139,8 +139,8 @@ int main(int argc, char *argv[])
     parm.target->key = "target";
     parm.target->type = TYPE_STRING;
     parm.target->required = NO;
-    parm.target->label = _("Name of GCPs target location");
-    parm.target->description = _("Name of location to create or to read "
+    parm.target->label = _("Name of GCPs target project (location)");
+    parm.target->description = _("Name of project to create or to read "
                                  "projection from for GCPs transformation");
```
Comment (veroandreo) [inline raster/r.in.gdal/main.c:144]:
```suggestion
                                 "CRS from for GCPs transformation");
```

File: raster/r.in.gdal/main.c [hunk 2]
```diff
@@ -214,9 +214,9 @@ int main(int argc, char *argv[])
     flag_o = G_define_flag();
     flag_o->key = 'o';
     flag_o->label =
-        _("Override projection check (use current location's projection)");
-    flag_o->description = _(
-        "Assume that the dataset has same projection as the current location");
+        _("Override projection check (use current project's projection)");
+    flag_o->description =
+        _("Assume that the dataset has same projection as the current project");
```
Comment (veroandreo) [inline raster/r.in.gdal/main.c:219]:
```suggestion
        _("Override projection check (use current project's CRS)");
    flag_o->description =
        _("Assume that the dataset has same CRS as the current project");
```

File: raster/r.in.gdal/main.c [hunk 3]
```diff
@@ -979,7 +979,7 @@ int main(int argc, char *argv[])
                                               &proj_units, hSRS, 0) == 1) {
                     G_warning(
                         _("Unable to convert input map projection to GRASS "
```
Comment (veroandreo) [inline raster/r.in.gdal/main.c:981]:
```suggestion
                        _("Unable to convert input map CRS to GRASS "
```

File: raster/r.in.gdal/main.c [hunk 4]
```diff
@@ -1133,21 +1133,20 @@ static void SetupReprojector(const char *pszSrcWKT, const char *pszDstLoc,
 
         /* Get projection info from target location */
         if ((out_proj_info = G_get_projinfo()) == NULL)
-            G_fatal_error(
-                _("Unable to get projection info of target location"));
+            G_fatal_error(_("Unable to get projection info of target project"));
```
Comment (veroandreo) [inline raster/r.in.gdal/main.c:1136]:
```suggestion
            G_fatal_error(_("Unable to get CRS info of target project"));
```

File: raster/r.in.gdal/main.c [hunk 5]
```diff
@@ -1133,21 +1133,20 @@ static void SetupReprojector(const char *pszSrcWKT, const char *pszDstLoc,
...
         if ((out_unit_info = G_get_projunits()) == NULL)
             G_fatal_error(
-                _("Unable to get projection units of target location"));
+                _("Unable to get projection units of target project"));
```
Comment (veroandreo) [inline raster/r.in.gdal/main.c:1139]:
```suggestion
                _("Unable to get CRS units of target project"));
```

File: raster/r.in.gdal/main.c [hunk 6]
```diff
@@ -1133,21 +1133,20 @@ static void SetupReprojector(const char *pszSrcWKT, const char *pszDstLoc,
...
         if (pj_get_kv(oproj, out_proj_info, out_unit_info) < 0)
             G_fatal_error(
-                _("Unable to get projection key values of target location"));
+                _("Unable to get projection key values of target project"));
```
Comment (veroandreo) [inline raster/r.in.gdal/main.c:1142]:
```suggestion
                _("Unable to get CRS key values of target project"));
```

Comment (veroandreo) [reply raster/r.in.gdal/main.c:1142]: Not sure if CRS key values makes sense though

File: raster/r.in.gdal/proj.c [hunk 7]
```diff
@@ -197,13 +197,13 @@ void check_projection(struct Cell_head *cellhd, GDALDatasetH hDS, char *outloc,
             int i_value;
 
             strcpy(error_msg, _("Projection of dataset does not"
```
Comment (veroandreo) [inline raster/r.in.gdal/proj.c:199]:
```suggestion
            strcpy(error_msg, _("Dataset CRS does not"
```

File: raster/r.in.gdal/proj.c [hunk 8]
```diff
@@ -322,10 +322,10 @@ void check_projection(struct Cell_head *cellhd, GDALDatasetH hDS, char *outloc,
                 strcat(error_msg, _("\nIn case of no significant differences "
                                     "in the projection definitions,"
```
Comment (veroandreo) [inline raster/r.in.gdal/proj.c:323]:
```suggestion
                                    "in the CRS definitions,"
```

File: raster/r.in.gdal/proj.c [hunk 9]
```diff
@@ -322,10 +322,10 @@ void check_projection(struct Cell_head *cellhd, GDALDatasetH hDS, char *outloc,
...
                                     " use the -o flag to ignore them and use"
-                                    " current location definition.\n"));
+                                    " current project definition.\n"));
                 strcat(error_msg, _("Consider generating a new location from "
```
Comment (veroandreo) [inline raster/r.in.gdal/proj.c:326]:
```suggestion
                strcat(error_msg, _("Consider generating a new project from "
```

File: raster/r.in.gdal/proj.c [hunk 10]
```diff
@@ -343,7 +343,7 @@ void check_projection(struct Cell_head *cellhd, GDALDatasetH hDS, char *outloc,
                 msg_fn = G_message;
             else
                 msg_fn = G_verbose_message;
-            msg_fn(_("Projection of input dataset and current location "
+            msg_fn(_("Projection of input dataset and current project "
```
Comment (veroandreo) [inline raster/r.in.gdal/proj.c:346]:
```suggestion
            msg_fn(_("CRS of input dataset and current project "
```

File: scripts/r.import/r.import.py [hunk 11]
```diff
@@ -106,8 +106,8 @@
 # %end
 # %flag
 # % key: o
-# % label: Override projection check (use current location's projection)
-# % description: Assume that the dataset has the same projection as the current location
+# % label: Override projection check (use current project's projection)
+# % description: Assume that the dataset has the same projection as the current project
```
Comment (veroandreo) [inline scripts/r.import/r.import.py:110]:
```suggestion
# % label: Override projection check (use current project's CRS)
# % description: Assume that the dataset has the same coordinate reference system (CRS) as the current project
```

**Review by veroandreo — COMMENTED** [review]
It was merged while I was reviewing... I have the same comment than in #3576, the use of projection and project in the same sentence sounds odd, CRS is more general than projection for that matter

**Pull request merged**